src/test/java/seedu/address/logic/parser/DeletePersonFromModuleCommandParserTest.java

```diff
@@ -0,0 +1,87 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_MODULE_CODE_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_NAME_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.MODULE_CODE_DESC_CS2106;
+import static seedu.address.logic.commands.CommandTestUtil.MODULE_CODE_DESC_MA2001;
+import static seedu.address.logic.commands.CommandTestUtil.NAME_DESC_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.NAME_DESC_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.PREAMBLE_WHITESPACE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_CS2106_MODULE_CODE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_AMY;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.DeletePersonFromModuleCommand;
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.person.Name;
+
+public class DeletePersonFromModuleCommandParserTest {
+
+    private DeletePersonFromModuleCommandParser parser = new DeletePersonFromModuleCommandParser();
+
+    @Test
+    public void parse_validArgs_returnsDeletePersonFromModuleCommand() {
+        String userInput = MODULE_CODE_DESC_CS2106 + " " + NAME_DESC_AMY;
```

Comment:
```suggestion
        String userInput = MODULE_CODE_DESC_CS2106 + NAME_DESC_AMY;
```
Hmm, I believe the `" "`  in the above line and also in the next two tests may not be needed since `NAME_DESC_AMY` is already prefixed with a `" "`.
![image](https://user-images.githubusercontent.com/83915879/197859388-377be4e5-54ba-485f-a00d-bc45fba20f2d.png)

Reply:
Thanks for spotting this! I shall update it now!